integration/getting_started_with_grafana_mimir_test.go

```diff
@@ -26,8 +26,12 @@ func TestGettingStartedWithGrafanaMimir(t *testing.T) {
 	defer s.Close()
 
 	require.NoError(t, copyFileToSharedDir(s, "docs/configurations/demo.yaml", "demo.yaml"))
+	flags := map[string]string{
+		// Enable protobuf format so that we can use native histograms.
+		"-query-frontend.query-result-response-format": "protobuf",
```

Comment:
[non blocking] I can see you're enabling it in many tests. Given this is required to get native histograms work, what if we just enable it by default in `integration/configs.go`?

Reply:
In the future, having it as a default might make sense, but for now, I'd like to leave it out so I can be confident I haven't broken the existing JSON format. WDYT?